CONTRIBUTING.md

```diff
@@ -21,3 +21,32 @@ Luckily, if you're not willing to get too involved with the technicalities of Gi
 The [Article Styling Criteria (ASC)](https://github.com/ppy/osu-wiki/tree/master/wiki/Article_styling_criteria) is a set of standards that must be adhered to for all new and edited articles going forward. Among these are rules regarding project structure, usage of Markdown, formatting, diction and grammar. Please refer to it if you have any specific questions or concerns.
 
 The ASC is fairly verbose and encompasses many minute details, that are not necessary to memorise in order to take the first step of contributing to the osu! wiki. Have a look at the [contribution guide](https://github.com/ppy/osu-wiki/tree/master/wiki/osu!_wiki/Contribution_guide), and other contributors will also be happy to answer your questions in the `#osu-wiki` channel of the [osu!dev Discord server](https://discord.gg/ppy).
+
+## Testing
+
+All pull requests are automatically run through a series of continuous integration checks (CI) in order to catch various common errors.
+
+For people familiar with the command line, these CI checks can also be run locally.
+
+### Requirements
+
+- Python 3.X
+- Node.js 16
+
+### Setup
+
+```sh
+pip3 install venv
+python3 -m venv ./.venv
+source ./.venv/bin/activate
```

Comment:
Is this actually required? It doesn't look well maintained and fails on non-bash shells:

```log
DEPRECATION: Configuring installation scheme with distutils config files is deprecated and will no longer work in the near fu
ture. If you are using a Homebrew or Linuxbrew Python, please see discussion at https://github.com/Homebrew/homebrew-core/iss
ues/76621
ERROR: Could not find a version that satisfies the requirement venv (from versions: none)                                    ERROR: No matching distribution found for venv                                                                               ./.venv/bin/activate (line 40): Unsupported use of '='. In fish, please use 'set VIRTUAL_ENV "/Users/dean/Projects/osu-wiki/.
venv"'.                                                                                                                      from sourcing file ./.venv/bin/activate                                                                                      source: Error while reading file './.venv/bin/activate'
```

If we *are* using this then it needs a `.gitignore` rule.

Reply:
I'll wait for @TicClick's opinion on this one

Reply:
I forgot that `venv` is a built-in module (unlike `virtualenv`), so the first one is my mistake.

as for the scripts, running `activate` and the likes is a matter of convenience if you don't want to specify the path to the environment's interpreter every time. it's still available under `./.venv/bin/python3`, and if this is intended to be a copy-pastable command, we may as well just use the direct path instead of a shell's discovery mechanisms.

(also, it has scripts for non-Bash shells, `activate.csh` and `activate.fish`)